index.js

```diff
@@ -47,5 +50,3 @@ internalIp.v6.sync = () => sync('v6');
 internalIp.v4.sync = () => sync('v4');
 
 module.exports = internalIp;
-// TODO: Remove this for the next major release
-module.exports.default = internalIp;
```

Comment:
@BendingBender is this correct? Do we need to update something else?

Reply:
Yes: https://github.com/sindresorhus/internal-ip/blob/5491cbc3447b51e0f818fe6086bc530ed35e338e/index.d.ts#L49-L50

Reply:
Done